### PR TITLE
Document the ability to define admin password for local deployment

### PR DESCRIPTION
**Pull request by tomekpapiernik**
**Description**

Changes proposed in this pull request:

- Document the ability to define admin password for local deployment
- Add cross-document linking in "Installation" documents
- Update "local installation scripts deep dive" doc with an up-to-date snippet of `run.sh` script
- Other minor tweaks and fixes. 

**Related issue(s)**
#1488

### CODE REVIEW

File: docs/kyma/docs/04-03-local-installation-from-sources.md [hunk 1]
```diff
@@ -17,14 +17,16 @@ To start the local installation, run the following command:
...
 
+- `--password {YOUR_PASSWORD}` which allows you to set a password for the **admin@kyma.cx** user.
 - `--skip-minikube-start` which skips the execution of the `installation/scripts/minikube.sh` script.
 - `--vm-driver` which points to either `virtualbox` or `hyperkit`, depending on your operating system.
 
-Read the **Reinstall Kyma** document to learn how to reinstall Kyma without deleting the cluster from Minikube.
-To learn how to test Kyma, see the **Testing Kyma** document.
+Read [this](#installation-reinstall-kyma) document to learn how to reinstall Kyma without deleting the cluster from Minikube.
+
+To learn how to test Kyma, see [this](#installation-reinstall-kyma) document.
```
Comment (klaudiagrz) [inline docs/kyma/docs/04-03-local-installation-from-sources.md]: Are you sure it's the right link? It lead to `Reinstall Kyma`, not to any testing scenario.

File: docs/kyma/docs/04-12-local-installation-scripts.md [hunk 2]
```diff
@@ -46,21 +45,21 @@ Once Minikube is up and running, the script adds local installation entries to `
 
 ## The build-kyma-installer.sh script
 
-The Installer is an application based on a [Kubernetes operator](https://coreos.com/operators/). Its purpose is to install Helm charts defined in the Installation custom resource. The Kyma-Installer is a Docker image that bundles the Installer binary with Kyma charts.
+The Kyma Installer is an application based on [Kubernetes operator](https://coreos.com/operators/). Its purpose is to install Helm charts defined in the Installation custom resource. The Kyma Installer is a Docker image that bundles the Installer binary with Kyma charts.
```
Comment (klaudiagrz) [inline docs/kyma/docs/04-12-local-installation-scripts.md]:
```suggestion
The Kyma Installer is an application based on the [Kubernetes operator](https://coreos.com/operators/). Its purpose is to install Helm charts defined in the Installation custom resource. The Kyma Installer is a Docker image that bundles the Installer binary with Kyma charts.
```

File: docs/kyma/docs/04-02-local-installation-from-release.md [hunk 3]
```diff
@@ -94,8 +94,9 @@ To install Kyma, follow these steps:
   ./scripts/is-installed.sh
   ```
 
-Read the **Reinstall Kyma** document to learn how to reinstall Kyma without deleting the cluster from Minikube.
-To learn how to test Kyma, see the **Testing Kyma** document.
+Read [this](#installation-reinstall-kyma) document to learn how to reinstall Kyma without deleting the cluster from Minikube.
+
+To learn how to test Kyma, see [this](#installation-reinstall-kyma) document.
```
Comment (klaudiagrz) [inline docs/kyma/docs/04-02-local-installation-from-release.md]: Are you sure it's the right link? It leads to `Reinstall Kyma`, not to any testing scenario.

File: docs/kyma/docs/04-02-local-installation-from-release.md [hunk 4]
```diff
@@ -142,13 +143,13 @@ http://192.168.64.44:30000
...
 ## Enable Horizontal Pod Autoscaler (HPA)
 
-By default, the Horizontal Pod Autoscaler is not enabled in your local Kyma installation, so you need to enable it manually.
+By default, the Horizontal Pod Autoscaler (HPA) is not enabled in your local Kyma installation, so you need to enable it manually.
 
-Kyma uses the autoscaling/v1 stable version, which only provides support for CPU autoscaling. Once enabled, HPA automatically scales the number of lambda function Pods based on observed CPU utilization.
+Kyma uses the `autoscaling/v1` stable version, which supports only CPU autoscaling. Once enabled, HPA automatically scales the number of lambda function Pods based on the observed CPU utilization.
 
->**NOTE:** The autoscaling/v1 version does not support custom metrics. To use such metrics, you need the autoscaling/v2beta2 version.
+>**NOTE:** The `autoscaling/v1` version does not support custom metrics. To use such metrics, you need the `autoscaling/v2beta2` version.
```
Comment (piotrmsc) [inline docs/kyma/docs/04-02-local-installation-from-release.md]: @Demonsthere shouldn't be v2beta1?

Comment (Demonsthere) [reply docs/kyma/docs/04-02-local-installation-from-release.md]: Hi, this is kubernetes version related. `v2beta2` is available only from `v1.12` https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#horizontalpodautoscaler-v1-autoscaling

**Pull request merged**